src/app/backend/client/manager.go

```diff
@@ -188,20 +204,57 @@ func (self *clientManager) buildAuthInfoFromConfig(cfg *rest.Config) api.AuthInf
 	}
 }
 
-// Extracts authentication information from request header
-func (self *clientManager) extractAuthInfo(req *restful.Request) api.AuthInfo {
+// Based on auth info and rest config creates client cmd config.
+func (self *clientManager) buildCmdConfig(authInfo *api.AuthInfo, cfg *rest.Config) clientcmd.ClientConfig {
```

Comment:
To be consistent I should remove comment because most of our private functions are not commented :smile:. If we don't have to add method name I wouldn't do that as it doesn't improve documentation and usually makes it harder to read.